.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,93 @@
+name: "🐛 Bug report"
+description: Report a bug found while using the Berty apps or packages.
+labels: ["bug", "Ready for Review"]
+assignees:
+- peletron
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: dropdown
+    id: product
+    attributes:
+      label: Berty product
+      description: What Berty product are you seeing the problem on?
+      options:
+        - Mobile app
+        - Desktop app
+        - CLI tools (berty mini, daemon, etc.)
+        - Code package
+    validations:
+      required: true
+  - type: input
+    id: product-version
+    attributes:
+      label:  Berty product version
+      description: What is the version of this Berty product?
+      placeholder: e.g. 2.396.1
+    validations:
+      required: true
+  - type: dropdown
+    id: os
+    attributes:
+      label: OS
+      description: What OS are you seeing the problem on?
+      multiple: true
+      options:
+        - iOS
+        - Android
+        - Linux
+        - macOS
+        - Windows
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: OS version
+      description: What is the version of the OS you are using?
+      placeholder: ex. iOS 15.4 or Android 11.0 or Ubuntu 20.04.4
+    validations:
+      required: true
+  - type: input
+    id: device
+    attributes:
+      label: Device
+      description: What devices are you using?
+      placeholder: e.g. iPhone 12 Pro Max or a synology NAS
```

Comment:
```suggestion
      description: What device are you using?
      placeholder: e.g. iPhone 12 Pro Max or a Synology NAS
```

Reply:
And if you want to use `e.g.` you would have to replace the other `ex.` in the form to be consistent.

Reply:
I switched everything to e.g. :)